### ISSUE TITLE
"Access (status) violation" on slicing anything 2.5.59.0 - no crash with console

### ISSUE THREAD

**Issue by aka13-404** [supermerill/SuperSlicer#3063]
### What happened?

I use my calicat.stl which has worked for many versions before, and try to slice it. 
I get  "Access (status) violation" and no crash_log.txt
![image](https://user-images.githubusercontent.com/51682059/184438944-d1e63a42-3968-4497-b332-a03ff22f58d2.png)
This is a clean install with configs freshly made, most stuff is on default. 
I get no crash when I run the same stl with the console version, so it must be the gui crashing? Not sure, what I may be doing wrong.
![image](https://user-images.githubusercontent.com/51682059/184440261-66220215-f28f-4196-b03c-5fc02919986a.png)

Appreciate your time as always, supermerill.

### Project file & How to reproduce

[Calibration Models.zip](https://github.com/supermerill/SuperSlicer/files/9329665/Calibration.Models.zip)

Simply import the stl, and press slice, you should get a crash. 

### Version

2.5.59.0

### Operating system

Windows 10

**Comment by Arakon** [comment]
Has to be a specific setting, slices fine here with my printer's profile..

![image](https://user-images.githubusercontent.com/12016360/184441942-5c2bb977-2284-472c-9994-ffc8e0c27bdf.png)

**Comment by aka13-404** [comment]
I figured that much myself, that it is propably something in the settings, but how comes, that it crashes only in the gui mode? 
Must be some gui pereference, that is causing the issue, that would be the only reasonable assumption in that case. 

Do you mind sharing your gui preferences tab, I'll just copy them and see if anything changes.

**Comment by aka13-404** [comment]
I can not reproduce it anymore, don't ask me how that works.

I have done the following steps: 
1) I backed up my config
2) I ran the setup wizard with some random printer
3) I compared the diff between the files, and changed them one after the other.

Results:
1) It was not the gui settings under perferences

2) Something along the highlighted options were responsible. I changed this whole section at once, so no idea what exactly it was. I tried setting it back to exactly how it is on the screenshot, but the crashing did not start again
![image](https://user-images.githubusercontent.com/51682059/184483914-fa750172-83f4-43a0-b3a9-fa04489ea3cd.png)

Conclusion:
It was some 1/0 magic, sadly could not reliably reproduce. Closing the issue.
